From ba53059d07007c3b6d100be93d0783e47843bf3c Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Mon, 8 Sep 2014 14:31:37 -0400
Subject: [scsi] hpsa: remove unused struct request from CommandList

Message-id: <1410186734-19986-47-git-send-email-thenzl@redhat.com>
Patchwork-id: 92080
O-Subject: [RHEL6.6 e-stor PATCH 46/83] hpsa: remove unused struct request from CommandList
Bugzilla: 1069185
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

Signed-off-by: Stephen M. Cameron <scameron@beardog.cce.hp.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
(cherry picked from commit c14c5891ddbd20cffbc4e35bfba8b7fbac9d5ada)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/hpsa_cmd.h b/drivers/scsi/hpsa_cmd.h
index 8026d2e..50388f1 100644
--- a/drivers/scsi/hpsa_cmd.h
+++ b/drivers/scsi/hpsa_cmd.h
@@ -386,7 +386,6 @@ struct CommandList {
  int      cmd_type;
  long      cmdindex;
  struct list_head list;
- struct request *rq;
  struct completion *waiting;
  void   *scsi_cmd;
 
@@ -399,8 +398,8 @@ struct CommandList {
  */
 #define IS_32_BIT ((8 - sizeof(long))/4)
 #define IS_64_BIT (!IS_32_BIT)
-#define PAD_32 (36)
-#define PAD_64 (4)
+#define PAD_32 (40)
+#define PAD_64 (12)
 #define COMMANDLIST_PAD (IS_32_BIT * PAD_32 + IS_64_BIT * PAD_64)
  u8 pad[COMMANDLIST_PAD];
 };
-- 
1.7.1